### PR TITLE
feat(#229) - Backend integration for home-spa and home-service

### LINKED ISSUE #229: Backend integration for home-spa and home-service
## Describe the solution you'd like

- The home-spa needs to fetch the microservice information form the backend

## Describe alternatives you've considered

None

### PR DESCRIPTION
**Pull request by hybridx**
feat(#229) - Integrated home-spa with graphql home-service

### Resolves #229 

# Explain the feature/fix

- Fetches SPA and microservice information form the home-service using the listHomeType query

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?

### CODE REVIEW

File: packages/home-spa/.env.example [hunk 1]
```diff
@@ -1 +1 @@
-API_URL=
\ No newline at end of file
+API_URL=https://localhost:8080/graphql
```
Comment (sayak-sarkar) [inline packages/home-spa/.env.example:1]: Why? I'd suggest keeping this:
```suggestion
API_URL=
```

Comment (hybridx) [reply packages/home-spa/.env.example]: Noted and changd

File: packages/home-spa/src/js/controller.js [hunk 2]
```diff
@@ -26,9 +26,10 @@ if (microserviceCard !== null && carouselSlide !== null) {
 
 getData().then(
   (data) => {
-    buildDom(data);
+    buildDom(data.data.listHomeType);
```
Comment (deshmukhmayur) [inline packages/home-spa/src/js/controller.js:29]: Change the argument name to something else, like `result` or something, as the data is being repeated.

Comment (hybridx) [reply packages/home-spa/src/js/controller.js]: Change applied

**Pull request merged**